scripts/shared/nodeinfo/nodeinfo.json

```diff
@@ -7,15 +7,15 @@
         "uri": "https://tomesh.net"
     },
     "software": {
-        "repo": "REPO",
-        "branch": "BRANCH",
-        "commit": "COMMIT",
-        "installed": "INSTALLED",
+        "repo": "__REPO__",
+        "branch": "__BRANCH__",
+        "commit": "__COMMIT__",
+        "installed": "__INSTALLED__",
         "uri": "https://github.com/tomeshnet/prototype-cjdns-pi/"
     },
-    "key": "KEY",
-    "org": "ORG",
+    "key": "__KEY__",
```

Comment:
Making cjdns first class makes it difficult to add another router / identity (e.g. yggdrasil, libp2p) in the future. Perhaps:

```
"routers": {
    "cjdns": {
        "key": "__KEY__"
    }
}
```

We don't have to address this now, since this is prototype and we can make breaking changes. Perhaps we need to version this JSON though.

Reply:
My bigger question is along side node exporter

How much do we want to share

Reply:
Created issue #173

Reply:
@benhylau Good point. I was trying to stick to the standard outlined [here](https://docs.meshwith.me/cjdns/nodeinfo-json.html) to some degree, but there's not a good reason why we have to.